### DISCUSSION TITLE
Optionally enable caching for extension bundles

### DISCUSSION THREAD

**Discussion by claytongulick** [directus/directus#14498] [Feature Requests]
Just a simple enhancement to speed loading times by optionally enabling cache-control for extension index.js bundles.

Added EXTENSIONS_CACHE_TTL env option.

PR incoming.

**Comment by claytongulick** [comment]
https://github.com/directus/directus/pull/14499

**Reply by claytongulick** [reply]
@rijkvanzanten Docs are in a separate repo - not sure if I should submit a PR to update them when I'm not sure this PR will be merged?

**Reply by rijkvanzanten** [reply]
PR was already merged 🙂 An update to the docs would be much appreciated!

**Reply by claytongulick** [reply]
Will do, sorry about the delay, at a conference this week. I’ll get a PR out as soon as I get my head above water.

**Reply by rijkvanzanten** [reply]
Godspeed out there! 🏊🏻

**Comment by rijkvanzanten** [comment]
Heya! Thank you for taking the time to submit this request! 

It has been over **90 days**, and this discussion has not received at least **15 votes** from the community. This means that we don't feel like there's enough community interest to warrant further R&D into this topic at this time. :ice_cube:

This request will now be closed to keep our discussions tidy. Please reach out if you have any questions!

_For more information, see our [Feature Request Process](https://docs.directus.io/contributing/feature-request-process.html)._
